components/Api/Stability/index.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import Stability from './index';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+type Story = StoryObj<typeof Stability>;
+type Meta = MetaObj<typeof Stability>;
+
+export const Stability0: Story = {
```

Comment:
I feel that only one Story might be enough? And allowing the user to select both the children and the `stability` number from a dropdown. (That can be done through the `select` control (https://storybook.js.org/docs/react/essentials/controls#annotation)

Reply:
I'll take a look for this 😁

Reply:
What do you thinks about my change ?